### PR TITLE
Hanoi

### PR DESCRIPTION
**Pull request by VictorBay24**
Adding Vietnam-Hanoi to the travel guide

Answers UCL-COMP0233-24-25/RSE-Classwork#3

### CODE REVIEW

**Comment by PyrgosRuairi** [comment]
You didnt link the issue!

File: asia/README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # ASIA
+
+[Vietnam](./Vietnam)
```
Comment (dpshelio) [inline asia/README.md:3]: This needs to link to the `README.md` file, no the folder.

**Review by dpshelio — CHANGES_REQUESTED** [review]
Almost perfect. You need to update a link, and then I'll suggest updating the title of the PR with something more meaningful (eg. "Hanoi") and add the issue this PR refers to on the body of the PR as instructed.

**Review by dpshelio — APPROVED** [review]
:tada:

**Pull request merged**